<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1962</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>920</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>TSC ERP Prototype Ver.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../ERP/logo.png</normaloff>../ERP/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
     <widget class="QStackedWidget" name="mainStackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1962</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1962</width>
        <height>900</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="mainStackedWidgetPage1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QScrollBar" name="verticalScrollBar">
        <property name="geometry">
         <rect>
          <x>860</x>
          <y>0</y>
          <width>16</width>
          <height>160</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1962</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QMenu" name="menu_Sales">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>영업모듈</string>
    </property>
    <widget class="QMenu" name="menu_SO">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>영업오더</string>
     </property>
     <addaction name="act_SO1"/>
     <addaction name="act_SO2"/>
     <addaction name="act_SO3"/>
    </widget>
    <addaction name="menu_SO"/>
    <addaction name="action_Customer"/>
    <addaction name="action_Project"/>
   </widget>
   <widget class="QMenu" name="menu_Prod">
    <property name="title">
     <string>생산모듈</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>생산오더</string>
     </property>
     <addaction name="action_Polist"/>
     <addaction name="action_Newpo"/>
     <addaction name="action3_2"/>
    </widget>
    <addaction name="act_PP"/>
    <addaction name="menu_4"/>
   </widget>
   <widget class="QMenu" name="menu_MM">
    <property name="title">
     <string>자재모듈</string>
    </property>
    <widget class="QMenu" name="menu_MC">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>자재코드</string>
     </property>
     <addaction name="act_MM03"/>
     <addaction name="act_MM01"/>
     <addaction name="act_MM02"/>
    </widget>
    <addaction name="menu_MC"/>
    <addaction name="action_Material"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>품질모듈</string>
    </property>
    <widget class="QMenu" name="menu_Record">
     <property name="title">
      <string>작업일보(수기)</string>
     </property>
     <addaction name="act_Search"/>
     <addaction name="act_Upload"/>
    </widget>
    <widget class="QMenu" name="menu_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>성적서</string>
     </property>
     <addaction name="act_MTR_Search"/>
     <addaction name="act_MTR_Publish"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="menu_Record"/>
    <addaction name="act_Data"/>
    <addaction name="separator"/>
    <addaction name="act_Lot"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>경영지원모듈</string>
    </property>
   </widget>
   <addaction name="menu_Sales"/>
   <addaction name="menu_Prod"/>
   <addaction name="menu_MM"/>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_SO1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>조회</string>
   </property>
  </action>
  <action name="act_SO2">
   <property name="text">
    <string>생성</string>
   </property>
  </action>
  <action name="act_SO3">
   <property name="text">
    <string>수정</string>
   </property>
  </action>
  <action name="action1_3">
   <property name="text">
    <string>조회</string>
   </property>
  </action>
  <action name="action2_3">
   <property name="text">
    <string>생성</string>
   </property>
  </action>
  <action name="action2_4">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="act_Release">
   <property name="text">
    <string>제품출하</string>
   </property>
  </action>
  <action name="act_MM03">
   <property name="text">
    <string>조회</string>
   </property>
  </action>
  <action name="act_MM01">
   <property name="text">
    <string>생성</string>
   </property>
  </action>
  <action name="act_MM02">
   <property name="text">
    <string>수정</string>
   </property>
  </action>
  <action name="act_PP">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>계획오더</string>
   </property>
  </action>
  <action name="act_Data">
   <property name="text">
    <string>치수데이터 업로드</string>
   </property>
  </action>
  <action name="act_Search">
   <property name="text">
    <string>조회</string>
   </property>
  </action>
  <action name="act_Upload">
   <property name="text">
    <string>업로드</string>
   </property>
  </action>
  <action name="act_MTR_Search">
   <property name="text">
    <string>조회</string>
   </property>
  </action>
  <action name="act_MTR_Publish">
   <property name="text">
    <string>발행</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>등록</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>조회</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>수정</string>
   </property>
  </action>
  <action name="action_Polist">
   <property name="text">
    <string>조회</string>
   </property>
  </action>
  <action name="action_Newpo">
   <property name="text">
    <string>생성</string>
   </property>
  </action>
  <action name="action3_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>수정</string>
   </property>
  </action>
  <action name="action_Customer">
   <property name="text">
    <string>거래처</string>
   </property>
  </action>
  <action name="act_Lot">
   <property name="text">
    <string>Lot Number</string>
   </property>
  </action>
  <action name="action_Project">
   <property name="text">
    <string>프로젝트 리스트</string>
   </property>
  </action>
  <action name="action_Material">
   <property name="text">
    <string>자재현황</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
